example/zoom/index.html

```diff
@@ -17,10 +17,11 @@
         <script src="../../dist/wavesurfer.js"></script>
         <script src="../../dist/plugin/wavesurfer.timeline.js"></script>
         <script src="../../dist/plugin/wavesurfer.regions.js"></script>
+        <script src="../../dist/plugin/wavesurfer.markers.js"></script>
```

Comment:
i would like to keep the zoom example as clean as possible (and remove other plugins in the future).

Reply:
ok, I'll pull it out.  was really just using it to test that my code reacted to zooming/scrolling properly.